bundles/org.openhab.binding.venstarthermostat/src/main/java/org/openhab/binding/venstarthermostat/internal/handler/VenstarThermostatHandler.java

```diff
@@ -174,6 +178,16 @@ public void handleCommand(ChannelUID channelUID, Command command) {
                 log.debug("Setting system mode to  {}", value);
                 setSystemMode(value);
                 updateIfChanged(CHANNEL_SYSTEM_MODE_RAW, new StringType("" + value));
+            } else if (channelUID.getId().equals(CHANNEL_AWAY_MODE)) {
+                VenstarAwayMode value;
+                if (command instanceof StringType) {
+                    value = VenstarAwayMode.valueOf(((StringType) command).toString().toUpperCase());
+                } else {
+                    value = VenstarAwayMode.fromInt(((DecimalType) command).intValue());
+                }
+                log.debug("Setting away mode to  {}", value);
+                setAwayMode(value);
+                updateIfChanged(CHANNEL_AWAY_MODE_RAW, new StringType("" + value));
```

Comment:
```suggestion
                updateIfChanged(CHANNEL_AWAY_MODE_RAW, new StringType(value.toString()));
```

Reply:
yep. I had copied the previous few lines of code. I will update mine and the previous one.